sysutils/osxbom/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# My repo is ahead of the main one in terms of versions:
+github.setup        cooljeanius osxbom 0.0.5
+github.tarball_from archive
+categories          sysutils
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
+license             public-domain
+
+description         A free re-implementation of the NextSTEP/OSX lsbom \
+                    utility
+long_description    ${description}. Used by the PureDarwin project \
+                    to work with MacPorts binary packages.
+homepage            https://cauldrondevelopment.com/svn/${name}/trunk/
+
+checksums           rmd160  ae9fa1120123aa0f19d1a92bce869d530556bf19 \
+                    sha256  788c6650494eff481baebe1129335003301d9530c20ad61cb0ec013b628797ca \
+                    size    155958
+
+# probably not actually necessary to autoreconf, but whatever:
+use_autoreconf      yes
+autoreconf.args     -fvi -Wall
+
+# mostly just things picked up by trace mode:
+depends_build-append \
+                    port:gawk \
+                    port:grep \
+                    bin:nroff:groff \
+                    bin:gmkdir:coreutils \
```

Comment:
These can be dropped.

Reply:
Trace mode says otherwise; I like my ports to be tracemode-clean